manifests/server/database.pp

```diff
@@ -147,4 +148,9 @@
     setting => 'conn-lifetime',
     value   => $conn_lifetime,
   }
+
+  ini_setting { 'puppetdb_database_max_pool_size':
+    setting => 'maximum-pool-size',
```

Comment:
This should be set at least optionally, and not be set when the module has no setting defining it. Also, we are baking in the setting of 25 into the module, we should just undef it, and only set when the user has set it explicitly.

Unless we ant to go and do all the work for making this backwards compatible also, the duplicated default-in-module still exists though.